libraries/Microsoft.Bot.Builder.Dialogs/Choices/Channel.cs

```diff
@@ -42,6 +54,15 @@ public static bool SupportsSuggestedActions(string channelId, int buttonCnt = 10
                 case Connector.Channels.Webchat:
                     return buttonCnt <= 100;
 
+                // https://learn.microsoft.com/en-us/microsoftteams/platform/bots/how-to/conversations/conversation-messages?tabs=dotnet1%2Cdotnet2%2Cdotnet3%2Cdotnet4%2Cdotnet5%2Cdotnet#send-suggested-actions
+                case Connector.Channels.Msteams:
+                    if (conversationType == "personal")
+                    {
+                        return buttonCnt <= 3;

```

Comment:
Why there is a limitation on 3 buttons? Is it a limitation on 1:1 suggested actions?

Reply:
@ceciliaavila, if this is a designed limitation for SA, do we need to document it?

Reply:
Teams display up to three suggested actions. I added the link to the documentation in the code.